authenticity-collector/src/main.ts

```diff
@@ -7,7 +7,7 @@ import { Logger } from '@nestjs/common';
 
 async function bootstrap() {
 	const app = await NestFactory.create(AppModule);
-
+	app.enableCors();
```

Comment:
pls remove this here and from the other services

Reply:
Removed, thanks for reminding me about that.